adafruit_rsa/key.py

```diff
@@ -45,58 +54,51 @@ class AbstractKey(object):
 
     __slots__ = ("n", "e")
 
-    def __init__(self, n, e):
+    def __init__(self, n: int, e: int) -> None:
         self.n = n
         self.e = e
 
     @classmethod
-    def _load_pkcs1_pem(cls, keyfile):
+    def _load_pkcs1_pem(cls, keyfile: bytes) -> "AbstractKey":
         """Loads a key in PKCS#1 PEM format, implement in a subclass.
 
-        :param keyfile: contents of a PEM-encoded file that contains
+        :param bytes keyfile: contents of a PEM-encoded file that contains
             the public key.
-        :type keyfile: bytes
-
         :return: the loaded key
         :rtype: AbstractKey
         """
 
     @classmethod
-    def _load_pkcs1_der(cls, keyfile):
+    def _load_pkcs1_der(cls, keyfile: bytes) -> "AbstractKey":
         """Loads a key in PKCS#1 PEM format, implement in a subclass.
 
-        :param keyfile: contents of a DER-encoded file that contains
+        :param bytes keyfile: contents of a DER-encoded file that contains
             the public key.
-        :type keyfile: bytes
-
         :return: the loaded key
         :rtype: AbstractKey
         """
 
-    def _save_pkcs1_pem(self):
+    def _save_pkcs1_pem(self) -> bytes:
         """Saves the key in PKCS#1 PEM format, implement in a subclass.
 
-        :returns: the PEM-encoded key.
+        :return: the PEM-encoded key.
         :rtype: bytes
         """
 
-    def _save_pkcs1_der(self):
+    def _save_pkcs1_der(self) -> bytes:
         """Saves the key in PKCS#1 DER format, implement in a subclass.
 
-        :returns: the DER-encoded key.
+        :return: the DER-encoded key.
         :rtype: bytes
         """
 
     @classmethod
-    def load_pkcs1(cls, keyfile, format="PEM"):
+    def load_pkcs1(cls, keyfile: bytes, format: str = "PEM") -> "AbstractKey":
```

Comment:
is `format` a good candidate for doing the `Literal["PEM", "DER"]` type? It looks like those are the two valid values from what I can tell.

Reply:
Yup!  Good catch!